ansible/playbooks/roles/repository/tasks/copy-download-requirements.yml

```
@@ -55,6 +55,11 @@
         dest: "{{ download_requirements_dir }}/{{ item }}"
       loop: "{{ _files }}"
 
+    - name: Copy the manifest file
+      synchronize:
```

```suggestion
      copy:
```
Isn't `synchronize` overkill for a single file?

---

Agreed ^

---

I don't understand why it's an overkill? Just because rsync has more options which I don't use here? Nevertheless it offers same speed for a small file. I just wanted to be consistent here and don't mix copy with synchronize.